### PR TITLE
Custom logo

### PR DESCRIPTION
**Pull request by asimqt**
[Custom Logo] Changes for implementing custom logo

### CODE REVIEW

File: app/assets/stylesheets/layout/_page-layout.scss [hunk 1]
```diff
@@ -135,7 +131,7 @@ body {
   }
 }
 
-.header__first__logo{
+.nl-logo__wrapper{
```
Comment (rvgpl) [inline app/assets/stylesheets/layout/_page-layout.scss]: what is `nl`? NewsLaundry?

Comment (asimqt) [reply app/assets/stylesheets/layout/_page-layout.scss]: Oops. My bad.

File: app/isomorphic/components/navigation-components/app-logo.js [hunk 2]
```diff
@@ -3,22 +3,18 @@ import classNames from 'classnames';
...
+function AppLogo(props) {
+  return <div className="app-logo">
+    <div className="app-logo__menu" onClick={props.openSidemenu}>
       <span className="menu-open"></span>
     </div>
-    <div className="header__first__logo">
-      <Link href="/">{props.title}</Link>
+    <div className="app-logo__wrapper">
+      <Link href="/">{
+        props.publisherTheme.logo ? <img className="app-logo__element" src={props.publisherTheme.logo} alt={props.title}/> :
```
Comment (vamshi29292) [inline app/isomorphic/components/navigation-components/app-logo.js]: Assign this to a variable, so JSX is clean.

File: app/isomorphic/components/navigation-components/offcanvas-menu.js [hunk 3]
```diff
@@ -29,7 +29,10 @@ class OffcanvasMenu extends React.Component {
         <nav className="sidebar__nav">
           <ul>
             <li className="sidebar__nav__logo" key={'nav-logo'}>
-              <Link href="/" className="logo logo--blue">{this.props.title}</Link>
+              <Link href="/" className="logo logo--blue">{
+                this.props.publisherTheme.logo ? <img className="app-logo__element" src={this.props.publisherTheme.logo} alt={this.props.title}/> :
```
Comment (vamshi29292) [inline app/isomorphic/components/navigation-components/offcanvas-menu.js]: Same as above.

**Pull request merged**